tests/e2e/test_execute_and_or.py

```
@@ -0,0 +1,81 @@
+from conftest import PROMPT, get_command_output
+
+
+def test_simple_and_success(shell_session):
+    shell_session.sendline("echo Hello && echo World")
+    shell_session.expect(PROMPT)
+
+    result = get_command_output(shell_session.before)
+    assert "Hello\nWorld" == result
```

Ruff

Yoda condition detected

Rewrite as `result == "Hello\nWorld"`

(SIM300)